sql/derived_tables/holdings_administrative_notes.sql

```diff
@@ -0,0 +1,25 @@
+-- This derived table extracts administrative notes from the instance
+-- record.
+
+DROP TABLE IF EXISTS holdings_administrative_notes;
+
+CREATE TABLE holdings_administrative_notes AS
+SELECT
+    h.id AS holdings_id,
+    h.hrid AS holdings_hrid,
+    admin_note.data #>> '{}' AS administrative_note,
+    admin_note.ordinality AS administrative_note_ordinality
+FROM
+    inventory_holdings AS h
+    CROSS JOIN LATERAL json_array_elements(json_extract_path(data, 'administrativeNotes'))
```

Comment:
We want lowercase instead of the camel case here with administrativeNotes.

Reply:
It is true that identifiers, such as names of tables and columns, should be in lowercase and should use underscores to separate words, rather than using camel case; this is documented in the contributing guidelines at https://github.com/folio-org/folio-analytics/blob/main/CONTRIBUTING.md#8-naming-things .  However, `'administrativeNotes'` is not an identifier but a string (note the surrounding `'`), and it refers to a JSON field; JSON fields do use camel case, and the camel case should be preserved when referring to them.